src/containers/BooksList.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Book from '../components/Book'
+import { removeBook } from '../actions'
+
+
+
+class BooksList extends React.Component {
```

Comment:
As I requested in Milestone 2 - BooksList isn't supposed to be a class component.